XenAdmin/ConsoleView/XSVNCScreen.cs

```diff
@@ -588,6 +589,29 @@ private void ConnectToRemoteConsole()
         void ConnectionSuccess(object sender, EventArgs e)
         {
             ConnectionRetries = 0;
+            if (AutoSwitchRDPLater)
+            {
+                if (OnDetectRDP != null)
+                    Program.Invoke(this, OnDetectRDP);
+                AutoSwitchRDPLater = false;
+            }
+            if (Helpers.CreamOrGreater(Source.Connection) && parentVNCTabView.IsRDPControlEnabled())
+                parentVNCTabView.EnableToggleVNCButton();
+        }
+
+        internal bool AutoSwitchRDPLater

```

Comment:
As no additional logic is not required here, this property can be auto-implemented, to make the declaration more concise, like:
internal bool AutoSwitchRDPLater { get; set; }